Tests/TuistKitTests/ProjectEditor/ProjectEditorMapperTests.swift

```diff
@@ -34,40 +34,55 @@ final class ProjectEditorMapperTests: TuistUnitTestCase {
-                                             product: .staticFramework,
-                                             productName: "Manifests",
-                                             bundleId: "io.tuist.${PRODUCT_NAME:rfc1034identifier}",
-                                             settings: expectedSettings(sourceRootPath: sourceRootPath),
-                                             sources: manifestPaths.map { (path: $0, compilerFlags: nil) },
-                                             filesGroup: .group(name: "Manifests"),
-                                             dependencies: [.target(name: "ProjectDescriptionHelpers")])
-        let expectedHelpersTarget = Target(name: "ProjectDescriptionHelpers",
-                                           platform: .macOS,
-                                           product: .staticFramework,
-                                           productName: "ProjectDescriptionHelpers",
-                                           bundleId: "io.tuist.${PRODUCT_NAME:rfc1034identifier}",
-                                           settings: expectedSettings(sourceRootPath: sourceRootPath),
-                                           sources: helperPaths.map { (path: $0, compilerFlags: nil) },
-                                           filesGroup: .group(name: "Manifests"),
-                                           dependencies: [])
-        let expectedProject = Project(path: sourceRootPath,
-                                      name: "Manifests",
-                                      settings: Settings(base: [:],
-                                                         configurations: Settings.default.configurations,
-                                                         defaultSettings: .recommended),
-                                      filesGroup: .group(name: "Manifests"),
-                                      targets: [expectedManifestsTarget, expectedHelpersTarget])
-        XCTAssertEqual(project, expectedProject)
-
         let targetNodes = graph.targets.sorted(by: { $0.target.name < $1.target.name })
         XCTAssertEqual(targetNodes.count, 2)
+        XCTAssertEqual(targetNodes.first?.dependencies, [targetNodes.last!])
+
+        // Generated Manifests target
+        let manifestsTarget = try XCTUnwrap(project.targets.first)
         XCTAssertEqual(targetNodes.first?.target, manifestsTarget)
+
+        XCTAssertEqual(manifestsTarget.name, "Manifests")
+        XCTAssertEqual(manifestsTarget.platform, .macOS)
+        XCTAssertEqual(manifestsTarget.product, .staticFramework)
+        XCTAssertEqual(manifestsTarget.settings, expectedSettings(sourceRootPath: sourceRootPath))
+        XCTAssertEqual(manifestsTarget.sources.map { $0.path }, manifestPaths)
+        XCTAssertEqual(manifestsTarget.filesGroup, .group(name: "Manifests"))
+        XCTAssertEqual(manifestsTarget.dependencies, [.target(name: "ProjectDescriptionHelpers")])
```

Comment:
Tried to check for things that make a difference in this case - but things like `bundleId` are not here.